### ISSUE TITLE
Releasing 1.8.0 

### ISSUE THREAD

**Issue by tisba** [processone/tsung#318]
Hi there.

I think it's time to think about cutting a 1.8.0 release of tsung! 🎉 

As time of writing, we have 43 commits by 11 contributors between master and develop: https://github.com/processone/tsung/compare/master...develop. It mainly contains bug fixes and some smallish features.

I created a [1.8.0 milestone](https://github.com/processone/tsung/milestone/4) which currently only contains three smallish things that would be nice to have covered, but I'm open to add or remove things.

I mainly create this issue to document what needs to be done for a release and make that documentation available.

**Comment by tisba** [comment]
Please chime in with your thoughts, @nniclausse. What do we need to do in order to get 1.8.0 out of the door? :)

**Comment by tisba** [comment]
While trying to make a release for the current develop to check if https://github.com/processone/tsung/issues/276 is fixed or not, I failed setting up a fresh Debian stretch VM so that `make release` works.

Is there some documentation available what packages (system, latex, perl, …) are required, @nniclausse?

**Comment by abregar** [comment]
Hello dear maintainers,
seems that progress is kind'a dormant on release next release. Is there any news on potential tag/release?
Keep up the good work!

**Comment by sashaafm** [comment]
I'm just commenting to say I'm also looking for a new release. At least to have a release with the commits that have been pushed to `develop` since `1.7.0`.

**Comment by nniclausse** [comment]
Yes, you are right, it's really time to publish a new release. I hope we can do it before the end of october

**Comment by tisba** [comment]
I'm happy to help as good as I can. But I'm quite swamped in work and also only use and know tsung for the HTTP specific parts.

**Comment by sashaafm** [comment]
Hello @tisba I guess this has long been forgotten? 🙂 any chance this project will be picked up again and release 1.8.0 (and eventually 2.0)?

**Comment by Neustradamus** [comment]
1.8.0 has been released:
- https://github.com/processone/tsung/releases/tag/v1.8.0
